goal_src/engine/ui/progress/progress.gc

```diff
@@ -75,7 +75,7 @@
   "Set the options for all of the menus."
 
   ;; start off by making them all invalid
-  (dotimes (i 35)
+  (dotimes (i 36)
```

Comment:
could you make this `(length *options-remap*)` instead, in case we add more in the future?

Reply:
can't getting the length on the options-remap returns `0`, perhaps this is a compiler bug on enums?
![image](https://user-images.githubusercontent.com/13153231/157166064-044c77b8-26c1-4b9c-a2ad-b42c718faeed.png)